tests/integration/test_experiments.py

```diff
@@ -0,0 +1,8 @@
+def test_list(client):
+
+    expected_experiments = ["test1", "test2", "test3"]
```

Comment:
Make this a set to begin with, rather than a list.
```suggestion
    expected_experiments = {"test1", "test2", "test3"}
```